portfolio/src/main/webapp/prototype.js

```diff
@@ -80,10 +80,27 @@ async function submitObject() {
     // If the load succeeded, display the analysis button.
     if (successful) {
         //ADD SOME PREVIEW HERE
-        document.getElementById(elements.analyzeButton).style.display = "block";
+        let request = new Request("/data?videoId=" + objectId, {method: "GET"});
```

Comment:
Right now if the user requested a playlist, this will try to get a video with the playlist's ID and will throw an error. Also, what's your plan for previewing playlists?